contracts/src/fp/OPSuccinctFaultDisputeGame.sol

```diff
@@ -304,7 +359,7 @@ contract OPSuccinctFaultDisputeGame is Clone, ISemver {
         if (msg.value != PROOF_REWARD) revert IncorrectBondAmount();
```

Comment:
The `challengeBond` is the prover's reward.

Proposer incentive: Don't create bad proposals.
Challenger incentive: Don't challenge proposals (otherwise you lose proof reward, which should be greater than cost to generate proof by significant amount).
Prover Incentive: Generate proofs if `PROOF_REWARD` is high enough.